docs/pages/components/autocomplete/examples/ExSimple.vue

```diff
@@ -8,6 +8,7 @@
                 :data="filteredDataArray"
                 placeholder="e.g. jQuery"
                 icon="magnify"
+                :clearable=true
```

Comment:
When is a boolean you can use like this:  

```suggestion
                clearable
```

Reply:
thanks :)